docs/github_actions.rst

```diff
@@ -17,64 +17,52 @@ Summary of Test Jobs
-| GCC8-MPI-CUDA-AFQMC-Complex-Mixed            | sulfur   | deterministic | 6    | manual   |
-+----------------------------------------------+----------+---------------+------+----------+
-| GCC8-MPI-CUDA-AFQMC-Real                     | sulfur   | deterministic | 6    | manual   |
-+----------------------------------------------+----------+---------------+------+----------+
-| GCC8-MPI-CUDA-AFQMC-Complex                  | sulfur   | deterministic | 6    | manual   |
-+----------------------------------------------+----------+---------------+------+----------+
-| Clang14Dev-MPI-CUDA-AFQMC-Offload-Real-Mixed | sulfur   | deterministic | 6    | manual   |
-+----------------------------------------------+----------+---------------+------+----------+
-| Clang14Dev-MPI-CUDA-AFQMC-Offload-Real       | sulfur   | deterministic | 6    | manual   |
-+----------------------------------------------+----------+---------------+------+----------+
-| Intel19-MPI-CUDA-AFQMC-Real-Mixed            | sulfur   | deterministic | 6    | manual   |
-+----------------------------------------------+----------+---------------+------+----------+
-| Intel19-MPI-CUDA-AFQMC-Complex-Mixed         | sulfur   | deterministic | 6    | manual   |
-+----------------------------------------------+----------+---------------+------+----------+
-| Intel19-MPI-CUDA-AFQMC-Real                  | sulfur   | deterministic | 6    | manual   |
-+----------------------------------------------+----------+---------------+------+----------+
-| Intel19-MPI-CUDA-AFQMC-Complex               | sulfur   | deterministic | 6    | manual   |
-+----------------------------------------------+----------+---------------+------+----------+
-| ROCm-Clang13-NoMPI-CUDA2HIP-Real-Mixed       | nitrogen | deterministic | 6    | manual   |
-+----------------------------------------------+----------+---------------+------+----------+
-| ROCm-Clang13-NoMPI-CUDA2HIP-Real             | nitrogen | deterministic | 6    | manual   |
-+----------------------------------------------+----------+---------------+------+----------+
-| ROCm-Clang13-NoMPI-CUDA2HIP-Complex-Mixed    | nitrogen | deterministic | 6    | manual   |
-+----------------------------------------------+----------+---------------+------+----------+
-| ROCm-Clang13-NoMPI-CUDA2HIP-Complex          | nitrogen | deterministic | 6    | manual   |
-+----------------------------------------------+----------+---------------+------+----------+
++---------------------------------------------------+----------+---------------+----------+----------+
+| Job Name with                                     | Runner   | Tests         | Time     | Trigger  |
+| Build Info                                        | Host     | ctest -L      | min      | event    |
++---------------------------------------------------+----------+---------------+----------+----------+
+| GCC9-MPI-Gcov-(Real*, Complex*)                   | GitHub   | deterministic | (60, 60) | PR/merge |
```

Comment:
Just bikeshedding, the * * is interpreted as italic for Complex.

Reply:
The rendered readthedocs page seems fine.